components/ui/ContentView.tsx

```diff
@@ -75,6 +80,13 @@ function ContentView({ post, isPreview = false, recycled }: ContentViewProps) {
 
   const onImagePress = () => {
```

Comment:
use `feedItem.setPostRead()` from `FeedItem.tsx`. Should just pass the function into ContentView with a prop.